go/integration_test/interpreter/stress_test.go

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package interpreter_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/Fantom-foundation/Tosca/go/interpreter/lfvm"
+	"github.com/Fantom-foundation/Tosca/go/tosca"
+	"github.com/Fantom-foundation/Tosca/go/tosca/vm"
+	"golang.org/x/sync/errgroup"
+)
+
+func BenchmarkParallelStressTests(b *testing.B) {
```

Comment:
This code does not really use the features of a benchmark, and the reported results when running this as a benchmark using `go test -bench` are likely not well defined. Maybe instead of writing this like a benchmark it should actually be a test.
```suggestion
func TestParallel_StressTests(b *testing.T) {
```

Not, when defining a benchmark, the parameter `b.N` defines a number of times the code under test should be processed and it has build-in parallel execution support (see [doc](https://pkg.go.dev/testing#hdr-Benchmarks)). However, if the benchmark is not making use of these facilities, the reported results of the benchmark tool are not accurate.

Reply:
I originally implemented it as a unit test, that's why I am using go's time function to measure runtime, but due to its extremely long runtime when running it for all interpreters I switched it to a benchmark. I rewrote it to make better use of go's benchmark infrastructure.